libcontainer/nsenter/nsexec.c

```diff
@@ -131,20 +140,40 @@ int setns(int fd, int nstype)
 }
 #endif
 
+void write_log_with_info(const char *level, const char *function, int line, const char *format, ...)
+{
+	static char message[1024];
```

Comment:
No need for this to be `static`. I would also do `= {}` just to be safe (initialisation-wise).

Reply:
Done